address issue with Glimma plots not appearing

grimbough opened the pull request:
This addresses #97 

The issue is that the `process_tables()` function adds the tag `<div class='horizontal-scroll'>` before any instances of `<table>`.  This inadvertantly breaks the Javascript that **Glimma** inserts into the page, by including some unexpected quotes.  Even if we change the quote type, adding this extra div is almost certainly not correct.

This patch means the additional div is only inserted around `<table>` tags that are part of the HTML body and not in the head.

grimbough committed:
address issue with Glimma plots not appearing

aoles commented on R/html_document.R:
```diff
@@ -294,8 +294,14 @@ process_footnotes = function(lines) {
+  body_start = grep(pattern = "<body>", x = lines, fixed = TRUE)
+  body_end = grep(pattern = "</body>", x = lines, fixed = TRUE)
+    
+  lines[body_start:body_start] = gsub("<table", "<div class='horizontal-scroll'><table", lines[body_start:body_start], fixed = TRUE)
```
I suspect a copy'n'paste' sort of bug here. Isn't the range supposed to read `body_start:body_end`?

aoles reviewed (changes requested):
Thanks a lot Mike @grimbough for providing the fix, and let me apologize for not getting back to you earlier 😇 

Please consider resolving the conflicts in **DESCRIPTION** and **NEWS** files, and my inline comment. Once these are addressed I'm happy to merge 🚀

grimbough committed:
Update outdated version number

Merge branch 'devel' of git.bioconductor.org:packages/BiocStyle into glimma-plots

# Conflicts:
#	DESCRIPTION
#	NEWS

grimbough committed:
Fix Copy/paste error

grimbough commented:
Hi @aoles 

Sorry, I forgot about this before the release.  I think I've made the appropriate changes for it do go into the current devel.
